skbio/io/format/tests/test_embedding.py

```diff
@@ -0,0 +1,121 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+import copy
+import io
+import string
+from unittest import TestCase, main
+from functools import partial
+from pathlib import Path
+import h5py
+import numpy as np
+import tempfile
+import skbio
+from skbio import Sequence, DNA, RNA, Protein, TabularMSA
+from skbio.embedding._protein import ProteinEmbedding
+from skbio.io import FASTAFormatError, QUALFormatError
+from skbio.io.format.embedding import (
+    _embed_sniffer, _embed_to_generator,
+    _embed_to_object, _generator_to_embed,
+    _objects_to_embed,
+    _embed_to_protein, _protein_to_embed
+)
+from skbio.util import get_data_path
+
+
+class EmbeddingTests(TestCase):
+    def setUp(self):
+        # single sequence
+        rk = 5  # latent dimension of residues
+        self.sequences = (
+            [
+                (
+                    np.load(get_data_path('embed1.txt.npy')),
+                    Protein(('IGKEEIQQRLAQFVDHWKELKQLAAARGQRL'
+                             'EESLEYQQFVANVEEEEAWINEKMTLVASED'),
+                            metadata={"id": "seq1"})
+                 ),
+                (
+                    np.load(get_data_path('embed2.txt.npy')),
+                    Protein(('QQNKELNFKLREKQNEIFELKKIAETLRSKL'
+                             'EKYVDITKKLEDQNLNLQIKISDLEKKLSDA'),
+                            metadata={"id": "seq2"})
+                )
+            ]
+        )
+        self.tempdir = tempfile.TemporaryDirectory()
```

Comment:
Could a `tearDown` be added which calls `self.tempdir.cleanup()`? I _think_ it's necessary to pass `ignore_cleanup_errors=True` to `TemporaryDirectory` as well

Reply:
Is that the case? `ignore_cleanup_errors=True` isn't being passed in the biom tests https://github.com/scikit-bio/scikit-bio/blob/main/skbio/io/format/tests/test_biom.py#L23

Reply:
It was necessary on another project, or possibly for windows support somewhere, but I can't remember exactly why right now

Reply:
I'm not able to pass this in.  I'm getting a `TypeError: __init__() got an unexpected keyword argument 'ignore_cleanup_errors' error with my version of python. Given that this isn't in the biom tests perhaps we can offload this to a future PR if necessarily